code/game/objects/items/flamethrower.dm

```diff
@@ -80,6 +80,8 @@
 		. += "+lit"
 
 /obj/item/flamethrower/ranged_interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+	if (!ptank || !lit)
+		return
```

Comment:
```suggestion
	if (!ptank)
		return
```
seemingly, `lit` is already handled on `flame_turf()`. also the logs below take `lit` being false or true into account when printing out statements so we'd miss out on that

Reply:
My bad, didn't realize that it gets handled in that.